src/mono/netcore/sample/wasm/Directory.Build.targets

```diff
@@ -0,0 +1,43 @@
+<Project>
+  <Import Project="$(MonoProjectRoot)\wasm\build\WasmApp.InTree.targets" />
```

Comment:
Could this be moved to the bottom? Or would that break things? I thought convention was to import props at the top and targets at the bottom.

Reply:
The convention is really for *proj files. The `*InTree*` props/targets are meant to work with the runtime repo, which has its own set of defaults that it sets, and in some order. So, we end up having to essentially work around that.